release-tools/keyrotation/key-rotation.py

```diff
@@ -0,0 +1,237 @@
+                {
+                'access_key_id': str(res.get('AccessKeyMetadata')[0].get('AccessKeyId')) ,
+                'create_date': res.get('AccessKeyMetadata')[0].get('CreateDate') 
+                })   
+            dict.append(
+                {
+                'access_key_id': str(res.get('AccessKeyMetadata')[1].get('AccessKeyId')) ,
+                'create_date': res.get('AccessKeyMetadata')[1].get('CreateDate')
+                })
+            logging.info(dict)
+            print(dict)
+    except:
+        logging.info("Failed to list keys for IAM user : " + iam_username)
+        print("Failed to list keys for IAM user : " + iam_username)
+    return dict
+
+# Delete the Access key for an IAM user
+def delete_key(iam_username,access_key):
+    try:
+        response = iam.meta.client.delete_access_key( 
+            UserName = iam_username,
+            AccessKeyId=access_key
+            )
+        logging.info(response)
+        print(response)
+    except:
+        logging.info("Failed to delete key : " + access_key + " For IAM user : " + iam_username)
+        print("Failed to delete key : " + access_key + " For IAM user : " + iam_username)
+
+# Deactiviate the Access key for an IAM user
+def deactivate_key(iam_username,access_key):
+    try:
+        response = iam.meta.client.update_access_key(
+            AccessKeyId=access_key,
+            Status='Inactive',
+            UserName=iam_username
+            )
+        logging.info("Response for the Access key deactiviation : " + str(response))
+        print("Response for the Access key deactiviation : " + str(response))
+    except:
+        logging.info("Failed to deactivate key : " + access_key + " For IAM user : " + iam_username)
+        print("Failed to deactivate key : " + access_key + " For IAM user : " + iam_username)
+
+# Get public key and key ID for Encrypting GIT secret
+def get_public_key(git_repo,git_owner):
+    try:
+        url = git_base_url + "/repos/" + git_owner + "/" + git_repo + "/actions/secrets/public-key"
+        logging.info("API URL for the public key : " + url)
+        print("API URL for the public key : " + url)
+        headers = {
+        "accept": "application/vnd.github.v3+json",
+        "authorization": git_auth_token
+        }
+        response=requests.get(url, headers=headers)
+        logging.info("Response for the public key request : ")
+        logging.info(response)
+        print("Response code : " + str(response))
+        logging.info("Public key for Repo : " + git_repo + " has been generated")
+        print("Public key for Repo : " + git_repo + " has been generated")
+    except:
+        logging.info("Failed to get public key for GIT owner : " + git_owner)
+        print("Failed to get public key for GIT owner : " + git_owner)
+    return response.json()
+
+
+# Update GIT secrets with the new AWS Access key ID and Access key Secret
+def update_git_secret(git_repo,git_owner,git_secret,secret):
```

Comment:
Maybe rename `git_secret` to `secret_name` to make it clearer what this argument means. I had to re-read this a couple of times to understand the difference between `git_secret` and `secret`.

Reply:
Will change it to git_secret_name.  Want to use "git" for all variables related to GIT.

Reply:
Change made to git_secret_name